tests/bignum.cc

```diff
@@ -0,0 +1,799 @@
+/*
+ *    Copyright 2022 Cryspen Sarl
+ *
+ *    Licensed under the Apache License, Version 2.0 or MIT.
+ *    - http://www.apache.org/licenses/LICENSE-2.0
+ *    - http://opensource.org/licenses/MIT
+ */
+#include <fstream>
+#include <gtest/gtest.h>
+#include <nlohmann/json.hpp>
+
+#ifdef HACL_CAN_COMPILE_VEC128
+#include "Hacl_Bignum256.h"
+#endif
+#include "Hacl_Bignum256_32.h"
+#include "Hacl_Bignum32.h"
+#ifdef HACL_CAN_COMPILE_VEC128
```

Comment:
Why don't you put all the `vec128` things into one guard?

Reply:
No reason. Simplified this bit, squashed, and force-pushed.